scripts-available/CDB_ZoomFromScale.sql

```diff
@@ -1,10 +1,9 @@
 CREATE OR REPLACE FUNCTION cartodb.CDB_ZoomFromScale(scaleDenominator numeric) RETURNS int AS $$
```

Comment:
This is coming from: https://github.com/CartoDB/CartoDB-basemaps/blob/7ca728b1a7b5d0d5c999c9eac1521e95eb5a4446/global_functions.sql#L77-L107

If we plan to change this, what about using https://github.com/mapnik/mapnik/wiki/ScaleAndPpi as reference to calculate the proper values?

See also: https://github.com/openstreetmap/mapnik-stylesheets/blob/236f8d0ab17b792300679cedf9afa5b0feab301b/zoom-to-scale.txt